packages/is-empty/index.js

```diff
@@ -0,0 +1,17 @@
+module.exports = isEmpty;
+
+/*
+ isEmpty({a: 3, b: 5}) // false
+
+ isEmpty(['a','b']) // false
+
+ isEmpty({}) // true
+
+ isEmpty([]) // true
+
+ isEmpty(null) // true
+*/
+
+function isEmpty(obj) {
+  return obj == null || (Array.isArray(obj) && !obj.length) || (typeof obj === 'object' && !Object.keys(obj).length);
```

Comment:
Can you break these into separate ifs. The reason is that a non empty array will currently unecessarily continue into the third test

if (obj == null) {
  return true
}
if (Array.isArray(obj) {
  return !obj.length
}
if (typeof obj == 'object' {
  return !Object.keys(obj).length;
}